lang/funcs/operator_polyfunc.go

```diff
@@ -38,6 +38,11 @@ const (
 	operatorArgName = "op" // something short and arbitrary
 )
 
+var epsilon = func() float64 {
```

Comment:
1) How can you simplify this?

2) It still runs for each use of the operator.

Can we do better?